lib/index.coffee

```diff
@@ -22,6 +27,7 @@ module.exports = (opts) ->
 
   # setup contentful api client
   client = contentful.createClient
+    host: hosts[process.env.CONTENTFUL_ENV] || hosts.production
```

Comment:
Could you change this to: `host: hosts[process.env.CONTENTFUL_ENV] || opts.host || hosts.production` - This will allow users to set the host variable themselves with multiple `app.coffee`s in roots.